Sebastian-Sole/handicappin
# ✨ feat: round calculation page

## What was done
- Created a page to view the calculations of rounds
- Interactive calculation of individual stats supported
- Users can view their rounds in detail with hole-by-hole statistics and the calculations done to affect their handicap

## How to Test
- Go to dashboard and click "view calculation" for a round
- See that the round data is displayed
- Test calculation fields

## Future Work
- View full calculator button
- Potential redesign
